### PR TITLE
Add Insecure flag to be able to connect to unsecured DNS server

### PR DESCRIPTION
**Pull request by broesch**

### CODE REVIEW

**Comment by vincent99** [comment]
LGTM, @janeczku?

File: providers/rfc2136/rfc2136.go [hunk 1]
```diff
@@ -33,18 +38,35 @@ func (r *RFC2136Provider) Init(rootDomainName string) error {
...
 	}
 
-	if keyName = os.Getenv("RFC2136_TSIG_KEYNAME"); len(keyName) == 0 {
-		return fmt.Errorf("RFC2136_TSIG_KEYNAME is not set")
+	if insecureStr := os.Getenv("RFC2136_INSECURE"); len(insecureStr) == 0 {
+		insecure = false
+	} else {
+		// if insecure, err := strconv.ParseBool(os.Getenv("RFC2136_INSECURE")); err != nil {
+		// insecure, err = strconv.ParseBool(os.Getenv("RFC2136_INSECURE"))
+		// if  err != nil {
```
Comment (janeczku) [inline providers/rfc2136/rfc2136.go]: Could you remove the leftover commented code?

**Comment by janeczku** [comment]
Thanks @broesch, this seems useful! See my comment above.

**Comment by broesch** [comment]
OK, I removed them.

**Comment by nheinemans** [comment]
We are currently running our own container from an internal catalog to use insecure DNS updates. We would like to move to the official Rancher catalog. When do you think this will get merged and released?

**Pull request merged**